app/over_react_redux/todo_client/pubspec.yaml

```
@@ -0,0 +1,20 @@
+name: todo_client
+version: 1.0.0
+
+environment:
+  sdk: ">=2.6.0 <3.0.0"
```

For extension methods, this would nee to be `>=2.7.0` wouldn't it?
```suggestion
  sdk: ">=2.7.0 <3.0.0"
```